warehouse/static/js/warehouse/controllers/notification_controller.js

```diff
@@ -46,7 +46,9 @@ export default class extends Controller {
     const notificationId = this._getNotificationId();
     if (notificationId) {
       localStorage.setItem(notificationId, 1);
+      this.notificationTarget.classList.remove("notification-bar--visible");
+    } else {
+      console.warn(`${this.notificationTarget} is dismissable but has no id attribute`); // eslint-disable-line no-console
```

Comment:
We support dismissable items without an ID I think? Those were the "emphereal" ones from my originally comment, and are things like flash messages which are going to go away on the next page load anyways (and so they don't need stored).

Reply:
Yeah, I forgot about that, fixing...